app/controllers/vacancies_controller.rb

```diff
@@ -40,7 +40,7 @@ def algolia_search_params
     %w[job_roles phases working_patterns].each do |facet|
       params[facet] = params[facet].split(" ") if params[facet].is_a?(String)
     end
-    params.permit(:keyword, :location, :location_category, :radius, :job_roles, :phases, :working_patterns,
+    params.permit(:keyword, :location, :location_category, :radius, :job_roles, :phases, :working_patterns, :buffer_radius,
```

Comment:
Is there an argument for storing both the size of the buffer and the size of the radius in one param? Because for the user, they are one and the same concept (they don't know we use polygons). Or does that make the code more complicated?

Reply:
We would interpret the param as a buffer radius if the search was in a polygon, and we would interpret it as a normal radius if the search was on a point.

Reply:
That would allow a job alert created before the introduction of polygons (or before the introduction of a specific polygon) to seamlessly continue having a radius.

E.g. in March 2018 I create a job alert for London with radius 10. In mid-2020, polygons are introduced, so my search criteria are interpreted as a search within London. The radius is discarded because I'm doing a polygon search now. So the search has become less relevant to me: it only searches London, not London + 10 miles.

Reply:
I think this is definitely outside the scope of the ticket. Also, a job alert created with `London + 10 miles` was created based on the centre of London + 10 miles, so the job alert is actually more relevant now rather than less relevant

Reply:
For the reasons Connor stated above, I'll leave this as it is for now.